src/main/java/mate.academy/model/Product.java

```diff
@@ -9,6 +9,9 @@ public class Product {
     private String description;
     private BigDecimal price;
 
+    public Product(String name, double price) {
+    }
```

Comment:
The constructor of 'Product' does not initialize the object's fields with the provided parameters. You should assign the 'name' and 'price' parameters to the corresponding fields to ensure the object is initialized correctly.